--- origsrc/libgtksourceviewmm-1.9.3/docs/reference/Makefile.am	2007-09-01 17:35:56.000000000 -0500
+++ src/libgtksourceviewmm-1.9.3/docs/reference/Makefile.am	2007-12-06 22:59:15.673250000 -0600
@@ -11,7 +11,7 @@
 cssfiles=$(shell ls -1 html/*.css)
 websitefiles=reference-index.txt
 
-docdir = $(datadir)/doc/libgtksourceviewmm-1.0/docs
+docdir = $(datadir)/doc/libgtksourceviewmm-2.0/docs
 htmldocumentationdir=$(docdir)/reference/html
 htmldocumentation_DATA=$(htmlfiles) $(pngfiles) $(giffiles) $(cssfiles)
 
--- origsrc/libgtksourceviewmm-1.9.3/tools/m4/Makefile.am	2007-09-01 17:35:56.000000000 -0500
+++ src/libgtksourceviewmm-1.9.3/tools/m4/Makefile.am	2007-12-06 22:52:26.360750000 -0600
@@ -5,6 +5,6 @@
 
 # Install the .m4, files:
 
-tools_m4_includedir = $(libdir)/gtksourceviewmm-1.0/proc/m4
+tools_m4_includedir = $(libdir)/gtksourceviewmm-2.0/proc/m4
 tools_m4_include_HEADERS = $(files_tools_m4)
 
